docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -931,3 +931,14 @@ spec:
   purpose: Informational
   tags: network, tls
   contributors: nabilschear
+---
+apiVersion: v1
+kind: query
+spec:
+  name: Discover Python Packages from Running Python Interpreters
+  platforms: Linux, macOS
+  description: Attempt to discover Python environments (in cwd, path to the python binary, and process command line) from running python interpreters and collect Python packages from those environments.
+  query: SELECT * FROM python_packages WHERE directory IN (SELECT DISTINCT directory FROM (SELECT SUBSTR(path,0,INSTR(path,'/bin/'))||'/lib' AS directory FROM processes WHERE path LIKE '%/bin/%' AND path LIKE '%python%' UNION SELECT SUBSTR(cmdline,0,INSTR(cmdline,'/bin/'))||'/lib' AS directory FROM processes WHERE cmdline LIKE '%python%' AND cmdline LIKE '%/bin/%' AND path LIKE '%python%' UNION SELECT cwd||'/lib' AS directory FROM processes WHERE path LIKE '%python%'))";
+  purpose: Informational
+  tags: compliance
```

```suggestion
  tags: compliance, hunting
```
Sweet query! I think we should add `hunting` as a label since it's more than a compliance thing. What do you think?